packages/products-used/style.module.css

```diff
@@ -0,0 +1,109 @@
+.root {
+  &.dark {
+    background-color: var(--wpl-neutral-900);
+  }
+}
+
+.eyebrow {
+  composes: g-type-label-strong from global;
+  margin: 0 0 var(--wpl-spacing-03) 0;
+  color: var(--wpl-neutral-500);
+
+  &.dark {
+    color: var(--wpl-neutral-400);
+  }
+}
+
+.productList {
+  display: flex;
+  align-items: center;
+  padding: 0;
+
+  & .isVerticallyCentered {
+    display: flex;
+    align-items: center;
+  }
+}
+
+.product {
+  &:not(:first-child) {
+    margin-left: var(--wpl-spacing-03);
+  }
```

Comment:
Can we rely on gap vs margins?

Reply:
One thing to flag is that products with links have different spacing between each other than products without links (8px and 16px respectively). Using `gap` would require that the parent that renders product items knows whether or not the items are links or non-links.

Not an issue per se but using gap does require some extra logic to expose this to the parent.

Reply:
Ahh I missed that so we do need the includesLink logic to be applied to the ul in some fashion 👍🏼. I think we'd still prefer this to be set via gap verses margin.